### PR TITLE
Fixes #13999: Cleanup reporting from rudder-techniques

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/13999

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

File: techniques/system/common/1.0/rudder-stdlib-core.cf [hunk 1]
```diff
@@ -1,285 +0,0 @@
...
-
-##################################################
-# classes bodies
-##################################################
-
-#
-# Automatically defines classes bases on a given prefix
-# The classes are defined based on the romises outcome
-#
-body classes rudder_common_classes(prefix)
```
Comment (peckpeck) [inline techniques/system/common/1.0/rudder-stdlib-core.cf]: this is used in many places

File: techniques/system/common/1.0/rudder-stdlib-core.cf [hunk 2]
```diff
@@ -1,285 +0,0 @@
...
-#    log_warn     (for logging only)
-#    log_info     (for logging only)
-#    log_debug    (for logging only)
-#    log_trace    (for logging only)
-# identifier     : the identifier of the current Rule and Directive
-# component_name : the name of the component within the Technique
-# component_key  : the value of the component reference variable (or None if undefined)
-# message        : An explanation message understandable by a human
-#
-bundle agent rudder_common_report(technique_name, status, identifier, component_name, component_key, message)
```
Comment (peckpeck) [inline techniques/system/common/1.0/rudder-stdlib-core.cf]: This one too

File: techniques/system/common/1.0/rudder-stdlib-core.cf (hunk 1 above)
Comment (peckpeck) [reply techniques/system/common/1.0/rudder-stdlib-core.cf]: aah migrated to ncf

Comment (amousset) [reply techniques/system/common/1.0/rudder-stdlib-core.cf]: Yes, this changes is parallel to https://github.com/Normation/ncf/pull/887

**Pull request merged**